portfolio/src/main/webapp/script.js

```diff
@@ -109,30 +109,90 @@ function getQuotes() {
 
   fetch(fetchUrl)
     .then(response => response.json())
-    .then((quotesJson) => {
-      const quoteListElement = document.getElementById('quoteWrapper');
-      quoteListElement.innerHTML = '';
-
-      Object.values(quotesJson).forEach(
-        (quote) => {
-          quoteListElement.appendChild(createQuotesListElement(quote)); 
-        });        
+    .then((responseJson) => {
+      updateLoginStatusElements(responseJson);
+      if (responseJson['Quote'] !== undefined) {
```

Comment:
I think you might be just able to remove the !== undefined part.

if (responseJson['Quote']) {
  ...
}

Reply:
I see. I also changed a few other parts where I was originally comparing variables to null or undefined.